src/ngMock/angular-mocks.js

```diff
@@ -2125,18 +2125,32 @@ angular.mock.$RootScopeDecorator = ['$delegate', function($delegate) {
 if (window.jasmine || window.mocha) {
 
   var currentSpec = null,
+      annotatedFunctions,
       isSpecRunning = function() {
         return !!currentSpec;
       };
 
+  var hookedAnnotate = angular.injector.$$annotate;
+  angular.injector.$$annotate = function(fn) {
+    if (typeof fn === 'function' && !fn.$inject) {
+      annotatedFunctions.push(fn);
+    }
+    return hookedAnnotate.apply(this, arguments);
```

Comment:
@lgalfaso I didn't hear back from you about this. Do you think this is ready to be merged?